### ISSUE TITLE
Assigning species to taxonomy

### ISSUE THREAD

**Issue by annabeth495** [ASSNAKE/assnake-dada2#1]
**assign_taxa.R** script assigns taxonomy up to genus, we would like to extend our pipeline by assigning taxonomy up to species, could you please add script which helps assign species taxonomy?

**Comment by Fedorov113** [comment]
Yeah, that feature is on our roadmap. I will add it in the coming days and let you know. 
If you have any other questions or feature requests - feel free to open an issue!
